container/broker/instance_broker.go

```diff
@@ -11,12 +11,12 @@ import (
 	"github.com/juju/names/v4"
 
 	"github.com/juju/juju/agent"
-	"github.com/juju/juju/api/common"
```

Comment:
If this import doesn't slap your face with "it shouldn't be there", I don't know what does.